Mathlib/Algebra/MvPolynomial/Eval.lean

```diff
@@ -490,6 +490,29 @@ theorem map_mapRange_eq_iff (f : R →+* S₁) (g : S₁ → R) (hg : g 0 = 0) (
   apply eq_iff_eq_cancel_right.mpr
   rfl
 
+lemma mem_range_map_of_coeffs_subset {f : R →+* S₁} {x : MvPolynomial σ S₁}
+    (hx : (x.coeffs : Set _) ⊆ Set.range f) : x ∈ Set.range (MvPolynomial.map f) := by
```

Comment:
I think this would be useful as an iff.

Reply:
And you can probably rephrase this as something like `(MvPolynomial.map f).range.toSubmodule =  MvPolynomial.coeffsIn 1` (not tested)